Docs(schlage): Add door code management actions (#40569)
home-assistant/home-assistant.github.io: source/_integrations/schlage.markdown

- **1-Touch Locking** - When enabled, locks the lock with a press of the Schlage button.
- **Keypress Beep** - Controls whether the lock will emit beeping tones on use.

## Actions

> ### Action schlage.get_codes

Reviewer: ```suggestion
### Action: Get codes
```
we use [this style for the heading](https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L139)

Author: Done

> You can use the `schlage.get_codes` action to retrieve the codes stored on your lock. The codes are returned as an object with all codes under the lock entity_id.

Reviewer: ```suggestion
You can use the `schlage.get_codes` action to retrieve the codes stored on your lock. This action returns all codes related to the Entity ID.
```
Unless it is really necessary to say that it returns an object, I would remove this.

Author: Done

{% details "Get codes action details" %}

| Data attribute | Optional | Description | Example |
| ---------------------- | -------- | ----------- | --------|
| `entity_id` | no | Lock entity to use (one or more) | `lock.front_door` |

{% raw %}

```yaml
# Example action
action: schlage.get_codes
data:
  entity_id:
    - lock.front_door
```

The returned codes will be in the following format:

```yaml
lock.front_door:
  93ab517c-0000-0000-0000-000000000000:
    name: Example Person
    code: "3333"
  82958b77-0000-0000-0000-000000000000:
    name: Example person 2
    code: "2222"
```

{% endraw %}

{% enddetails %}

### Action schlage.add_code

You can use the `schlage.add_code` action to add a new code to your lock. The code must be between 4 and 8 digits long.

{% details "Add code action details" %}

> | Data attribute | Optional | Description | Example |

Reviewer: we [prefer lists over tables](https://developers.home-assistant.io/docs/documenting/general-style-guide#tables)
https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L143

Author: When I look at other documentation that has actions, they are also using tables for the action details.

For instance `source/_integrations/zwave.markdown` and `source/_integrations/xiaomi_aqara.markdown`

When I try and use the list format that you pointed me at the preview docs do not render cleanly, but they do when using a table.
